evap/evaluation/migrations/0055_reviewer_group.py

```diff
@@ -2,11 +2,11 @@
 from django.db import migrations
 
 
-def add_group(apps, schema_editor):
+def add_group(_apps, _schema_editor):
```

Comment:
as this is not only about `schema_editor` could you please rename the commit to `Mark unused arguments in migrations`?